sample_files/20_input_files/ReportParameters.yml

```diff
@@ -0,0 +1,34 @@
+default:
+    Naming: 
+      GraphType: Report
+      Scenariochoose: SDS
+      AccountingPrinciple: PortfolioWeight
+      ScenarioGeographyChoose: GlobalAggregate
+      EquityMarketSelection: Global
+      Language: "EN"
+      Startyear: 2019
+      
+      
+    Benchmarks:
+      PeerGroupSelection: Project
+      EQPeersRef: Project
+      CBPeersRef: Project
+      eq_market_ref: ACWI	
+      cb_market_ref: BEHGTRUU   
+
+    Contents:
+      templateversion: "GeneralTemplateInput_v6"
+      BondReference: "Bond"
+      RefMarketData: TRUE
+      WithCompanyCharts: TRUE
+      IsSample: FALSE
+      IncWWF: FALSE
+      IncPRI: FALSE
+      IncCoalRetirement: TRUE
+      IncOtherSectors: TRUE
+      IncAlignment: TRUE
+      IncPeersChart: FALSE
+      IncPhysicalRisk: FALSE
+      inc_sda_approach: TRUE     
+      No.Companies: 15
```

Comment:
Nit: The style seems inconsistent. Best is to stick to one (say `snake_case`). A popular style guide is https://style.tidyverse.org/, which includes naming conventions.

Reply:
It most definitely is inconsistent, but I feel uncomfortable changing variable names that propagate through this repo. Which is a good sign this whole thing ought to get a re-write at some point.